lib/wallet/wallet.js

```diff
@@ -595,6 +595,7 @@ class Wallet extends EventEmitter {
       id: this.id,
       name: this.accountDepth === 0 ? 'default' : name,
       witness: options.witness,
+      lookahead: options.lookahead,
```

Comment:
To reviewers: This is the fix for https://github.com/bcoin-org/bcoin/issues/660

Reply:
Maybe this could be broken out into its own PR such that its easier to merge in smaller pieces

Reply:
Yeah, that one could be broken out, it's mostly necessary here to fix an issue in regards to syncing SPV nodes.